tutorials/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -307,8 +307,8 @@ editor. For example, when looking at a differential revision.
 One statement per line
 ~~~~~~~~~~~~~~~~~~~~~~
 
-Never combine multiple statements on a single line. No, C programmers,
-not even with a single line conditional statement.
+Avoid combining multiple statements on a single line, including conditional
+statements, to adhere to the GDScript style guidelines for readability.
```

Comment:
There is ternary operator, which can also be considered a conditional statement. I'm not sure if we should focus on it at all (since the example below demonstrates this case). If so, could we reformulate this to avoid a collision with the ternary operator?

Reply:
From what I understand, the exception for ternary operators is already addressed in the documentation, in lines 335-339 which is right after the examples.
Can you clarify what new collision this would create with the ternary operator?

I believe this change makes the style and the ternary operator exception clear
If there is something I am misunderstanding, can you propose the suggested edit?

Reply:
This is just my suggestion of what could be improved, but I don't think it's critical. You can leave it as is if you feel so.